app/serializers/v1/fee_serializer.rb

```diff
@@ -80,5 +80,15 @@ def taxes
     def legacy_values
       ::V1::Legacy::FeeSerializer.new(model).serialize
     end
+
+    def calculate_pay_in_advance
```

Comment:
What about simply renaming this one `pay_id_advance`?

Reply:
fixed